cmd/test/main.go

```diff
@@ -69,6 +73,12 @@ func (flags *TestCommandFlags) Validate() error {
 
 }
 
+func (flags *TestCommandFlags) Transform() {
```

Comment:
You can set the default value when you register the flag, for example, this set "simple" a the default value: 
```
testCommand.Flags().StringP("output", "o", "simple", "Define output format")
```
wrap it in an if clause and you got yourself a deal-krokodil